lib/java/AdapterLibrary/build.gradle.kts

```diff
@@ -98,6 +98,6 @@ publishing {
     }
 }
 
-signing {
-    sign(publishing.publications["mavenJava"])
-}
+//signing {
+//    sign(publishing.publications["mavenJava"])
+//}
```

Comment:
We'll need this to publish to Maven. It shouldn't prevent you from building locally - but either way, shouldn't be committed.

Reply:
my bad, forgot to revert this after testing locally